OpenTranslator/Controllers/Awesome/PascalTranslationController.cs

```diff
@@ -226,18 +226,20 @@ enum BlockType
         /// <param name="input"></param>
         /// <param name="LanguageCode"></param>
         /// <param name="filename"></param>
-        void LoadFromReader(TextReader reader, string input, string filename = "")
+        void LoadFromReader(TextReader reader, string input, string filename = "",string Languagecode="")
```

Comment:
method params should be in camelCase

Reply:
Yes, I have changed it to "languagecode"